spec/unit/provider/uri_parsing_spec.rb

```diff
@@ -380,6 +387,14 @@
       data = { 'osDeploymentPlanUri' => name }
       expect(uri_validation(data)).to eq 'osDeploymentPlanUri' => uri
     end
+
+    it 'should replace SAS Interconnect name by uri when key is SASInterconnectUri' do
```

Comment:
We can remove this test case as it doesn't cover the intended lines in the uri_parsing module.

Reply:
Reemoved